package.json

```diff
@@ -39,10 +39,16 @@
     "vscode-debugprotocol": "^1.11.0"
   },
   "devDependencies": {
-    "fs-extra": "^0.30.0",
-    "tslint": "^3.14.0",
-    "typescript": "1.8.10",
-    "vscode": "^0.11.17"
+    "@types/diff": "0.0.31",
+    "@types/fs-extra": "0.0.35",
+    "@types/lodash": "^4.14.40",
```

Comment:
The travis tests for linux are failing to find types/lodash. Am not sure why, but for now you can remove lodash from your refactoring